src/main/java/Calculator.java

```diff
@@ -0,0 +1,75 @@
+public class Calculator {
+
+    public static final String plus = "+";
+    public static final String minus = "-";
+    public static final String multiple = "*";
+    public static final String divide = "/";
```

Comment:
```suggestion
    public static final String PLUS = "+";
    public static final String MINUS = "-";
    public static final String MULTIPLE = "*";
    public static final String DIVIDE = "/";
```

상수의 경우 대문자로 작성해주는 것이 관례입니다 :)